core/audits/bf-cache.js

```diff
@@ -0,0 +1,126 @@
+/**
+ * @license Copyright 2022 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import {Audit} from './audit.js';
+import * as i18n from '../lib/i18n/i18n.js';
+import {NotRestoredReasonDescription} from '../lib/bfcache-strings.js';
+
+/* eslint-disable max-len */
+const UIStrings = {
+  /** Title of a diagnostic Lighthouse audit that identifies when the back/forward cache is being used. "back/forward" refers to the back and forward buttons found in modern browsers. This title is shown to users if the back/forward cache was used, or if the back/forward cache was not used for reasons outside the user's control. */
+  title: `Page didn't prevent back/forward cache restoration`,
```

Comment:
```suggestion
  title: `Page supports back/forward cache restoration`,
```

Reply:
I thought this title might be misleading if there are non-actionable reasons bf-cache is blocked.

However, saying that the *page* specifically supports it would work I think.

Reply:
A page doesn't support it though - it just didn't add anything to block it.

These things could be added at a later stage though (e.g. on an interaction, or lazy-loading some content further down the page, such as a commenting system), that the audit might not have triggered.

I think it might be safer sticking with the "Page didn't prevent back/forward cache restoration" wording even if linguistically Connor's suggest reads better.

Or the current dev tools audit uses "Successfully served from back/forward cache." Maybe something similar works here? "Page was successfully served from the back/forward cache."

Reply:
Keeping it as is sounds good (i.e. "Page didn't prevent back/forward cache restoration")